website/docs/compute/deployment/cloud-deployment/configure-storage.md

```diff
@@ -1,20 +1,78 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Configure storage
 
 By default, Compute will store compiled functions and function source code on the local storage in your Kubernetes cluster. For greater scalability, Compute can be configured to store artifacts in cloud-based object storage like Amazon S3 or Google Cloud Storage.
 
-To configure a storage bucket, provide the `SCC_STORAGE_PATH` environment variable to both the control plane and builder, e.g. `s3://my-bucket` for Amazon S3 or `gs://my-bucket` for Google Cloud Storage. For Kubernetes deployments, this is done in `.suborbital/scc-controlplane-deployment.yaml` under the `controlplane` and `builder` containers sections, and for local docker-compose deployments, this is done in `docker-compose.yaml` under the `scc-control-plane` and `scc-builder` services.
-
 ## Authentication
 
 Bucket authentication varies between cloud providers.
 
+<Tabs groupId='cloud-provider'>
+
+<TabItem value="S3" label="Amazon S3">
+
 ### Amazon S3
 
-You will need to also supply the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` and `AWS_REGION` environment variables to both the control plane and the builder. See the [AWS authentication documentation](https://docs.aws.amazon.com/sdk-for-go/api/aws/session/) for details. It is also possible to store the configuration as a Kubernetes secret, similar to the Google Cloud Storage configuration.
+You will need to supply the `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN` and `AWS_REGION` environment variables to the API for both the control plane and the builder. See the [AWS authentication documentation](https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials_environment.html) for details. It is also possible to store the configuration as a Kubernetes secret, similar to the Google Cloud Storage configuration.
```

Comment:
We should specify that they need to be added to the yaml file, and also show them in the example. I also wonder if we want to keep this bit about the Kubernetes secret. Maybe we have it link to the other tab?

Reply:
We do specify the different yaml files in the next paragraph down: 

> For Kubernetes deployments, this is done in `.suborbital/scc-controlplane-deployment.yaml` under the `controlplane` and `builder` containers sections, and for local docker-compose deployments, this is done in `docker-compose.yaml` under the `scc-control-plane` and `scc-builder` services.

> and also show them in the example

Do you mean show the file name in the yaml codeblock? I considered that but decided against it since there are two candidates for the file name depending on whether a deployment is on Kubernetes or local. 

> I also wonder if we want to keep this bit about the Kubernetes secret. Maybe we have it link to the other tab?

I'll add a link to the GCS tab!